src/sidebar/components/HelpPanel.tsx

```diff
@@ -114,7 +114,7 @@ function HelpPanel({ session }: HelpPanelProps) {
       onActiveChanged={onActiveChanged}
       variant="custom"
     >
-      <TabHeader>
+      <TabHeader closeTitle="Close help dialog">
```

Comment:
This is a non-modal "panel" rather than a "dialog". I suggest we use the term "panel" consistently for this type of widget.